node/src/accountant/mod.rs

```diff
@@ -1991,47 +1481,29 @@ mod tests {
         log_handler.exists_log_containing("WARN: Accountant: Encountered transaction error at this end: \
          'TransactionFailed { msg: \"Attempt failed\", hash_opt: Some(0x0000000000000000000000000000000000000000000000000000000000003039)");
         log_handler.exists_log_containing(
-            "DEBUG: Accountant: Deleting an existing fingerprint for a failed transaction 0x0000000000000000000000000000000000000000000000000000000000003039",
+            "DEBUG: Accountant: Deleting an existing backup for a failed transaction 0x0000…3039",
```

Comment:
Fingerprint is better word here...and it's the official term so please let it be "fingerprint". Remember this is a debug log which is intended for us developers and we should know what it is.

Also please change the hash to be full.